### PR TITLE
so many minutes so little time

### PR DESCRIPTION
**Pull request by joesepi**
Closes: #956
Closes: #952 
Closes: #946 
Closes: #940 
Closes: #934
Closes: #929 
Closes: #921 
Closes: #915 
Closes: #909
Closes: #904 
Closes: #893 
Closes: #883 
Closes: #870 
Closes: #864 
Closes: #834 
Closes: #831 
Closes: #828 
Closes: #822 
Closes: #819

### CODE REVIEW

File: meetings/2022/2022-10-11.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  * no news yet
+
+* OpenJS World 2023 planning [#908](https://github.com/openjs-foundation/cross-project-council/issues/908)
+  * program committee meeting this Thursday
+
+* CPC At Large voting member election [#906](https://github.com/openjs-foundation/cross-project-council/issues/906)
+  * closed
+
+* CPC Impact voting member selection [#905](https://github.com/openjs-foundation/cross-project-
+council/issues/905)
```
Comment (joesepi) [inline meetings/2022/2022-10-11.md:44]:
```suggestion
* CPC Impact voting member selection [#905](https://github.com/openjs-foundation/cross-project-council/issues/905)
```

**Review by WaleedAshraf — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Review by tobie — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

File: meetings/2022/2022-01-18.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+* Clarify requirements around CoC violation [#786](https://github.com/openjs-foundation/cross-project-council/issues/786)
+  * Have a working session planned for Feb 8
+
+* Create proposal(s) for the Community Fund [#756](https://github.com/openjs-foundation/cross-project-council/issues/756)
+  * see new repo: https://github.com/openjs-foundation/community-fund
+
+* List documents maintained by the CPC [#789](https://github.com/openjs-foundation/cross-project-council/pull/789)
+  * Tobie has removed the WIP prefix and draft status of this PR
+
+
```
Comment (ovflowd) [inline meetings/2022/2022-01-18.md:70]: I think you accidentally added some extra lines here?

File: meetings/2022/2022-01-18.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+
+## Next week's working session
```
Comment (joesepi) [inline meetings/2022/2022-01-18.md:73]:
```suggestion

## Next week's working session
```

**Review by eemeli — APPROVED** [review]

**Pull request merged**